test/resources/oc.resource

```diff
@@ -52,3 +52,10 @@ Oc Wait For
     ${timeout_opt}=    Set Variable    --timeout\=${timeout}
     ${output}=    Run With Kubeconfig    oc wait -n ${NAMESPACE} ${type_name} --for=${wait_for} ${timeout_opt}
     RETURN    ${output}
+
+Oc Create
+    [Documentation]    Run 'oc create' on a specific resource in the curret test namespace
+    ...    Returns the command's combined STDOUT/STDER
+    [Arguments]    ${opts}
```

Comment:
The format of arguments in this function is not consistent to the other similar functions in this file. 
Can we add concrete options, rather then one generic `opts`?